### ISSUE TITLE
obssigner.service does not start, as "sign program is not configured!"

### ISSUE THREAD

**Issue by johanneskastl** [openSUSE/open-build-service#14706]
# Issue Description
The obssigner.service does not start, as the "sign program is not configured!"

Expected Result
============
The service should start successfully. Or the [documentation](https://openbuildservice.org/help/manuals/obs-admin-guide/obs.cha.installation_and_configuration.html) should contain the steps that are necessary to configure it.

I guess there are steps in the setup-appliance.sh script that do $MAGIC in the background. Those steps are missing in the documentation.

How to Reproduce
===============
1. Add the 2.10 repo on a Leap 15.3 instance (not 15.4, which is not yet supported, as I found out)
2. Install the OBS packages
3. Try to start the obssigner.service

**Comment by pallaswept** [comment]
> I guess there are steps in the setup-appliance.sh script that do $MAGIC in the background.

Indeed. They also generate errors that the version of sign does not support required features, but at least it works if you install following the docs that say to use the setup script.
